inc/command/class-command.php

```diff
@@ -422,40 +430,63 @@ protected function codecept( InputInterface $input, OutputInterface $output ) {
 			);
 		}
 
-		$this->create_test_db( $input, $output );
+		if ( $test_suite ) {
+			$suites = [ $test_suite ];
+		} else {
+			// Codeception command runs within `vendor` directory, so paths are relative to that.
+			$suites = $this->get_test_suites( 'vendor/' . $tests_folder );
+			$output->write(
+				sprintf( '<info>Detected %d suites, (%s)..</info>', count( $suites ), implode( ', ', $suites ) ),
+				true,
+				$output::VERBOSITY_NORMAL
+			);
+		}
 
-		// Ensure cache is clean.
-		$this->run_command( $input, $output, 'wp', [ 'cache', 'flush' ] );
+		$return = '';
```

Comment:
Return value should be an integer exit code, so zero in this case if the command is considered successful so far.

Reply:
That inspired me to add a `-a` argument to continue running tests if one suite fails, and default to halting execution after a single failure.